browser/src/control/Control.NotebookbarBuilder.js

```diff
@@ -44,6 +44,7 @@ L.Control.NotebookbarBuilder = L.Control.JSDialogBuilder.extend({
 		this._toolitemHandlers['.uno:Save'] = this._saveControl;
 		this._toolitemHandlers['.uno:SaveAs'] = this._saveAsControl;
 		this._toolitemHandlers['.uno:Print'] = this._printControl;
+		this._toolitemHandlers['.uno:Settings'] = this._onlineHelpControl;
```

Comment:
ah that special widget I want to remove at some point, we have dispatcher now to share code with menubar, it was old hack to execute menubar action from notebookbar in the past :)